git-lfs.yaml

```diff
@@ -36,6 +36,9 @@ pipeline:
 
       # mitigate CVE-2022-27191
       go get golang.org/x/crypto/ssh
+
+      # mitigate GHSA-vvpx-j8f3-3w6h
+      go get golang.org/x/net@v0.7.0
```

Comment:
Do we need a `secfixes`/`advisory` entry for this? And the above?

Reply:
I don't think we do secfixes for non-CVEs, but I could be mistaken.

Reply:
Secfixes/advisories are only for vulns for `apk` packages, not `go` packages (i.e. modules). So no need to add anything here!